src/mapping.ts

```diff
@@ -16,6 +16,7 @@ export function loadOrCreateHolder(address: Address): Holder {
   let holder = Holder.load(address.toHex());
   if (!holder) {
     holder = new Holder(address.toHex());
+    holder.balance = BigInt.zero();
```

Comment:
So we have to initialize zeroes now?

Reply:
Yes, otherwise the indexing errors out. Seems to be a breaking Graph change.